### PR TITLE
NSX: Add passive monitor for NSX LB to test whether a server is available

### PR DESCRIPTION
**Pull request by Pearl1594**
### Description

This PR adds active and passive LB monitors to check the health of the server and prevent sending requests to faulty / unreachable servers.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

File: plugins/network-elements/nsx/src/main/java/org/apache/cloudstack/service/NsxApiClient.java [hunk 1]
```diff
@@ -111,6 +116,10 @@ public class NsxApiClient {
     protected static final String SEGMENTS_PATH = "/infra/segments";
     protected static final String DEFAULT_DOMAIN = "default";
     protected static final String GROUPS_PATH_PREFIX = "/infra/domains/default/groups";
+    // TODO: Pass as global / zone-level setting?
+    protected static final String NSX_LB_PASSIVE_MONITOR = "/infra/lb-monitor-profiles/default-passive-lb-monitor";
```
Comment (nvazquez) [inline plugins/network-elements/nsx/src/main/java/org/apache/cloudstack/service/NsxApiClient.java:120]: In that case, only this one should be a setting or other paths above as well?

Comment (Pearl1594) [reply plugins/network-elements/nsx/src/main/java/org/apache/cloudstack/service/NsxApiClient.java:120]: Honestly for everything, we'd need a better way of handling it that hard-coding it.

**Comment by Pearl1594** [comment]
@nvazquez please merge if you think it's good to go .. Thanks!

**Review by nvazquez — APPROVED** [review]

**Comment by nvazquez** [comment]
@blueorangutan package

**Comment by nvazquez** [comment]
@blueorangutan package

**Comment by nvazquez** [comment]
@blueorangutan package

**Pull request merged**